riak/datadog_checks/riak/data/conf.yaml.example

```diff
@@ -1,67 +1,90 @@
+## All options defined here are available to all instances.
+#
 init_config:
 
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
+
+## Every instance is scheduled independent of the others.
+#
 instances:
 
     ## @param url - string - required
     ## Riak stats url to connect to.
     #
   - url: http://127.0.0.1:8098/stats
 
-    ## @param min_collection_interval - integer - required - default: 60
-    ## Riak statistics are calculated on a per-minute sliding window,
-    ## Change the default collection interval if need be
-    #
-    min_collection_interval: 60
-
     ## @param cacert - string - optional
     ## Path to the TLS CA certificate file (ca.pem).
     #
     # cacert: <TLS_CA_CERT_PATH>
 
-    ## @param disable_cert_verify - boolean - optional - default: false
+    ## @param disable_cert_verify - boolean - optional - default: False
```

Comment:
Strange this gets output as `False` instead of `false`?

Reply:
Removing the `default` from the spec put it back to `false`, I'm not sure why